tests/ir/test_clone.py

```diff
@@ -223,6 +223,7 @@ def f(x, y):
 
 
 def test_clone_without_forcing_manager():
+    @clone
```

Comment:
So we clone three times now?

Reply:
There is a manager on the result of `@parse`. The `@clone` is basically doing `f = clone(f)`, which gives us a graph without a manager. The subsequent clones are cloning that clone.

Reply:
ok